sdcm/cluster_gce.py

```diff
@@ -366,56 +302,89 @@ def set_tags_as_labels(_instance):
             self._gce_services[dc_idx].connection.timeout = gce_job_default_timeout
         return instance
 
+    def create_instance_definition(self, data_disks: list[DataDisk], dc_idx, node_index):
```

Comment:
see `sdcm/sct_provision/instances_provider.py` and how instances definitions are build for Azure - there's just one place where we build it which is used during initial provision and during tests.